src/gpu/intel/ocl/micro_sdpa.cl

```diff
@@ -341,10 +345,18 @@ micro_sdpa(const global KEY_DATA_T *K, const global QRY_DATA_T *Q,
     if (k0end > 0) {
 #if WITH_ATTN_SCALE
 #if INVERT_SCALE
```

Comment:
We should be able to do the inversion on the host side as well. Ideally the host side scale should be able invert before the kernel is called as opposed to performing this operation on each work-item.